geostore/tests/test_models/test_other_models.py

```diff
@@ -34,3 +36,11 @@ def test_clean(self):
                                          })
         feature.clean()
         self.assertIsNotNone(feature.pk)
+
+    def test_feature_geom_3d(self):
```

Comment:
this is not so explicit. this should be named 'test_geom_3d_to_2d or something like this